include/settings/types/Velocity.h

```diff
@@ -12,74 +14,24 @@ namespace cura
  *
  * This is a facade. It behaves like a double, only it can't be negative.
  */
-struct Velocity
+struct Velocity : public utils::NumericFacade<double>
```

Comment:
The comment above says how 'double, only it cant be negative', but I don't see any mechanism taking care of that.

We should either fix that, or it's also a candidate to be replaced by typedef.